pkg/options/start.go

```diff
@@ -348,6 +348,32 @@ func (intOpts *InteractiveOpts) TaskWorkspaces(task *v1beta1.Task) error {
 	return nil
 }
 
+// Include the default param values in opts.Params when `--use-param-defaults` is used
```

Comment:
Would it make more sense to define this directly in `task/start.go` since it only is for tasks?

Reply:
Yup, makes sense, Thanks for the suggestion :)